scripts/install_pecan.sh

```diff
@@ -362,6 +362,9 @@ EOF
   rm /tmp/pecan.conf
 fi
 
+# adding the permissions to www-data to execute bash scripts which need root privilages
```

Comment:
This is bad and should not be done. www-data should NEVER be able to run as root.